TASVideos/Pages/Games/Versions/Edit.cshtml.cs

```diff
@@ -174,7 +174,7 @@ public async Task<IActionResult> OnPost()
 
 		return string.IsNullOrWhiteSpace(HttpContext.Request.ReturnUrl())
 			? RedirectToPage("List", new { gameId = GameId })
-			: BaseReturnUrlRedirect($"?GameId={GameId}&GameVersionId={version.Id}");
```

Comment:
if I leave this here it appends the game twice both times with `?` which breaks the selection syntax.